tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -25,8 +25,8 @@ module.exports = {
 
     assertSharingNotAllowed: function () {
       // eslint-disable-next-line no-unused-expressions
-      this.api.expect.element(this.elements.sharingAutoComplete.selector).not.to.be.present
-      return this.api.getText(this.elements.sharingSidebarRoot.selector,
+      this.api.expect.element(this.elements.addShareButton.selector).not.to.be.present
```

Comment:
that button seems not to exist anymore. its now `.files-collaborators-collaborator-add` not `#files-collaborators-add-new-button`

Reply:
there are actually two buttons: "@createShare" which matches "files-collaborators-collaborator-add" which is the button that triggers the share dialog. => now renamed to "files-collaborators-open-add-share-dialog-button"

the second one is "@addShareButton" which matches "files-collaborators-collaborator-add-new-button" and is actually the "Save" button in the share creation dialog => now renamed to "files-collaborators-collaborator-save-new-share-button"

I've renamed them to hopefully make this clearer.
Naming is hard...

Reply:
@individual-it pushed. Please see the last commit and resolve this conversation if the change is acceptable